website/en/docs/faq/index.md

```diff
@@ -143,6 +143,76 @@ const isNumber = (valueToRefine: ?number): %checks => typeof valueToRefine === '
 if (isNumber(val)) add(val, 2);
 ```
 
+### Why can't I pass Array<string> to function that takes Array<string | number>
+
+The function allows string values in an array, but in this case Flow protects the original array. In the function you would be able to push number to it which means that the type of original array would not be true. You can get rid of this by changing the argument to `$ReadOnlyArray<string | number>`. This prevents following code to push anything which makes it safe to pass stricter types to it.
+
+This would not work:
+
+```js
+// @flow
+
+const fn = (arr: Array<string | number>) => {
+	// arr.push(123) // note! Array<string> passed in and after this it would also contain numbers
+	return arr
+};
+
+const arr: Array<string> = ['abc'];
+
+fn(arr); // Error!
+```
+
+but with $ReadOnlyArray you can achieve what you were looking for:
+
+```js
+// @flow
+const fn = (arr: $ReadOnlyArray<string | number>) => {
+    // arr.push(321) //note! Array<string> passed in and after this it would also contain numbers
```

Comment:
I would update this comment, to say that since you're using a `$ReadOnlyArray`, we cannot push.